src/instruments/src/EFB/TODCalculator/GroundSpeed/GroundSpeedAuto/GroundSpeedAuto.tsx

```diff
@@ -16,21 +16,17 @@ const GroundSpeedAuto = ({ groundSpeedData, currentAltitude, setTodGroundSpeed,
     let [simGroundSpeed] = useSimVar('GPS GROUND SPEED', 'knots', 1_000);
     simGroundSpeed = round(simGroundSpeed);
 
-    const setCurrentGroundSpeed = () => {
+    const { groundSpeed } = last(groundSpeedData);
```

Comment:
While this PR doesn't really change this, this behaviour seems dodgy for two reasons:
- If you're storing data in a React component, that is state, and should be treated as such (useState over last)
- Storing past prop values can be dodgy for the same reasons as prevTime in the clock components above. I'm confused why this is even necessary in this component.